core/src/wsv.rs

```diff
@@ -418,9 +418,7 @@ impl<W: WorldTrait> WorldStateView<W> {
     /// **Locking behaviour**: Holding references to blocks stored in the blockchain can induce
     /// deadlock. This limitation is imposed by the fact that blockchain is backed by [`dashmap::DashMap`]
     #[inline]
-    pub fn blocks(
-        &self,
-    ) -> impl Iterator<Item = impl Deref<Target = VersionedCommittedBlock> + '_> {
```

Comment:
why this change?

Reply:
I need to use `rev()` method which is provided by `ChainIterator`

Reply:
I think that an opaque iterator here only makes things harder.

Reply:
`DoubleEndedIterator` can be used, but if you maintain that it reduces readability resolve this discussion